parts/kubelet-restart.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ ! -e /opt/azure/containers/runcmd.complete ]; then
+  exit 0
+fi
```

Comment:
Does `runcmd.complete`  refer to https://github.com/Azure/acs-engine/blob/master/parts/kubernetesagentcustomdata.yml#L158, which signifies the server has been provisioned? Want to make sure I understand the significance here.